tests/subsys/edac/ibecc/src/ibecc.c

```diff
@@ -310,9 +310,7 @@ static void test_ibecc_error_inject_test_uc(void)
 }
 #endif
 
-void test_edac_dummy_api(void);
-
-void test_main(void)
+void ibecc_before(void *data)
```

Comment:
I think this is `setup()` function

Reply:
This is strange, this code was running before all tests with older API, the same way setup() is supposed to run.